.github/workflows/dockerbuild.yml

```diff
@@ -0,0 +1,18 @@
+# This workflow will do a clean install of node dependencies, build the source code and run tests across different versions of node
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-nodejs-with-github-actions
+
+name: Docker build
+
+on:
+  push:
+    branches: [master]
+  pull_request:
+    branches: [master]
+
```

Comment:
I guess this workflow will run, when I create a pull request, that wants to merge sth to master. Will it also run, if I commit to the branch after I created the pull request?

Reply:
Yup, it will run on every commit to the pull request.